tests/misc/test_split.py

```diff
@@ -103,37 +103,39 @@
 áéíóú/áéíóú/áéíóú/
 """
 
+test_data_lines = test_data.strip().splitlines()
 
-class SplitTests(unittest.TestCase):
+
+class TestSplit:
 
     """Test split."""
 
-    def test_split(self):
+    @pytest.mark.parametrize('cmd, out',
+                             [case.split('/')[:-2]
+                              for case in test_data_lines])
```

Comment:
Hm, what about "pre-processing" the test data to create namedtuples? i.e. something like

```python
test_data_str = """
...
"""

TestCase = collections.namedtuple('TestCase', 'input, keep, no_keep')

test_data = []
for line in test_data_str.splitlines():
    data = line.split('/')
    item = TestCase(input=data[0], keep=data[1].split('/'),
                    no_keep=data[2].split('/'))
    test_data.append(item)
```

And then using the data like this in the tests? The question is where to do that pre-processing, seems a bit much code to make it all global - would it be a good idea to use a fixture for that?